lib/percy.rb

```diff
@@ -66,6 +66,11 @@ def self._get_agent_js
 
   def self._make_dom_snapshot(page)
     agent_js = self._get_agent_js
+
+    if self._is_debug?
+      self._logger.info { "agent_js file: #{agent_js}" }
```

Comment:
Added this log because it's been really helpful to know what this is while debugging various integrations with customers.